docs/reference/commandline/run.md

```diff
@@ -58,8 +58,8 @@ parent = "smn_cli"
                                     'bridge': creates a new network stack for the container on the docker bridge
                                     'none': no networking for this container
                                     'container:<name|id>': reuses another container network stack
-                                    'host': use the host network stack inside the container
-                                    'NETWORK': connects the container to user-created network using `docker network create` command
+                                    'host': uses the host network stack inside the container
+                                    'NETWORK_NAME_or_ID': connects the container to user-defined network using `docker network create` command
```

Comment:
I'm a bit in doubt if we should be this explicit, or just use "NETWORK", and mention that either the name or id of the network can be used.

@moxiegirl wdyt?